Lib/test/test_gdb.py

```diff
@@ -161,8 +161,11 @@ def get_stack_trace(self, source=None, script=None,
         if (gdb_major_version, gdb_minor_version) >= (7, 4):
             commands += ['set print entry-values no']
 
+        # bpo-32962: When Python is compiled with -mcet -fcf-protection,
+        # arguments are unusable before running the first instruction
+        # of the function entry point. Next makes the required first step.
```

Comment:
Would you mind to replace Next with >The 'next' command< to make it more explicit?

Move also the comment inside the if block.